Requirements/GassistPi-system-requirements.txt

```diff
@@ -12,5 +12,4 @@ python3-venv
 python3
 python3-pip
 socat
-mpg123
```

Comment:
Needed for "say" function (Text to speech converter).

Reply:
not need, I converted the say function to use pygame instead [L185!](https://github.com/moham96/GassistPi/blob/0212d52a9ab0282d746753e2e4f8972fe58b9816/src/actions.py#L185)